experiments/AtmosGCM/heldsuarez.jl

```diff
@@ -28,44 +29,29 @@ end
 function init_heldsuarez!(bl, state, aux, coords, t)
     FT = eltype(state)
 
-    # Parameters need to set initial state
-    T_ref = bl.data_config.T_ref
-    temp_profile = IsothermalProfile(T_ref)
-
-    # Calculate the initial state variables
-    T, p = temp_profile(bl.orientation, aux)
-    thermo_state = PhaseDry_given_pT(p, T, bl.param_set)
-    ρ = air_density(thermo_state)
-    e_int = internal_energy(thermo_state)
-    e_pot = gravitational_potential(bl.orientation, aux)
-
-    # Set initial state with random perturbation
+    # Set initial state to reference state with random perturbation
     rnd = FT(1.0 + rand(Uniform(-1e-3, 1e-3)))
-    state.ρ = ρ
+    state.ρ = aux.ref_state.ρ
     state.ρu = SVector{3, FT}(0, 0, 0)
-    state.ρe = rnd * state.ρ * (e_int + e_pot)
+    state.ρe = rnd * aux.ref_state.ρe
 
     nothing
 end
 
 function config_heldsuarez(FT, poly_order, resolution)
-    exp_name = "HeldSuarez"
-
-    # Parameters
-    T_ref::FT = 255
-    Rh_ref::FT = 0
-    domain_height::FT = 30e3
-    turb_visc::FT = 0 # no visc. here
-
-    # Set up a reference state for linearization
-    temp_profile_ref = IsothermalProfile(T_ref)
-    ref_state = HydrostaticState(temp_profile_ref, Rh_ref)
-
-    # Rayleigh sponge to dampen flow at the top of the domain
-    z_sponge = FT(15e3) # height at which sponge begins
-    α_relax = FT(1 / 60 / 15) # sponge relaxation rate in (1/seconds)
-    u_relax = SVector(FT(0), FT(0), FT(0)) # relaxation velocity
-    exp_sponge = 2 # sponge exponent for squared-sinusoid profile
+    # Set up a reference state for linearization of equations
+    T_sfc::FT = 290 # surface temperature of reference state (K)
+    ΔT::FT = 60     # temperature drop between surface and top of atmosphere (K)
+    H_t::FT = 8e3   # height sclae over which temperature drops (m)
+    temp_profile_ref = DecayingTemperatureProfile(T_sfc, ΔT, H_t)
+    ref_state = HydrostaticState(temp_profile_ref, FT(0))
+
+    # Set up a Rayleigh sponge to dampen flow at the top of the domain
+    domain_height::FT = 30e3               # distance between surface and top of atmosphere (m)
+    z_sponge::FT = 12e3                    # height at which sponge begins (m)
```

Comment:
This is an uncomfortably low-reaching sponge. Can you run this with `z_sponge = 20e3`?

Reply:
I don't think the simulation is stable in that case. I can try and increase the strength, but I tried many variations and sponges that are shallow don't work at this point.